cmd/gpu_plugin/README.md

```diff
@@ -208,66 +98,20 @@ $ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes
 $ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/nfd/overlays/node-feature-rules?ref=<RELEASE_VERSION>'
 
 # Create GPU plugin daemonset
-$ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/gpu_plugin/overlays/monitoring_shared-dev_nfd/?ref=<RELEASE_VERSION>'
+$ kubectl apply -k 'https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/gpu_plugin/overlays/nfd_labeled_nodes?ref=<RELEASE_VERSION>'
```

Comment:
> Does operator (have an option to) enable it?

It has an option but the default CR doesn't have it enabled. Operator by nature is more configurable compared to yaml deployments. Though, I do think that monitoring should be enabled by default.

I'd prefer not using words like "normal" because it means different things for different people. Even "basic" may be thought to be different.

Reply:
Good point about the names.  I noticed you decided to enable monitoring for both. Otherwise maybe following titles would have been more acceptable:
* Minimal install with NFD
* Configurable install with Operator